EIPS/eip-1.md

```diff
@@ -114,7 +114,8 @@ Each EIP should have the following parts:
 EIP Formats and Templates
 -------------------------
 
-EIPs should be written in [markdown] format. Image files should be included in a subdirectory for that EIP.
+EIPs should be written in [markdown] format.
+Image files should be included in a subdirectory of the `assets` folder for that EIP as follow: `assets/eip-X` (for eip *X*).
```

Comment:
Two * for bold.

Reply:
I thought it looked nicer italics, but I'm happy to make it bold.